### PR TITLE
Update python examples in gsets.md

### PR DESCRIPTION
**Pull request by JMercerGit**
Updated examples in gsets.md for the python sections to reflect the gsets functionality.

### CODE REVIEW

File: content/riak/kv/2.2.6/developing/data-types/gsets.md [hunk 1]
```diff
@@ -450,19 +440,19 @@ var_dump($gset->getData());
...
 
 ```python
-cities_set.dirty_value
+gset.dirty_value
 
 # The value fetched from Riak is always immutable, whereas the "dirty
 # value" takes into account local modifications that have not been
 # sent to the server. For example, where the call above would return
 # frozenset(['Toronto', 'Hamilton', 'Ottawa']), the call below would
 # return frozenset([]).
```
Comment (Bob-The-Marauder) [inline content/riak/kv/2.2.6/developing/data-types/gsets.md]: The above text (lines 445-449) needs to be adjusted to the current banking context

File: content/riak/kv/2.2.6/developing/data-types/gsets.md [hunk 2]
```diff
@@ -450,19 +440,19 @@ var_dump($gset->getData());
...
 
-cities_set.value
+gset.value
 
 # To fetch the value stored on the server, use the call below. Note
 # that this will clear any unsent additions or deletions.
```
Comment (Bob-The-Marauder) [inline content/riak/kv/2.2.6/developing/data-types/gsets.md]: Deletions are not possible in gsets

Comment (JMercerGit) [reply content/riak/kv/2.2.6/developing/data-types/gsets.md]: Adjusted to remove reference to deletions

File: content/riak/kv/2.2.6/developing/data-types/gsets.md (hunk 1 above)
Comment (JMercerGit) [reply content/riak/kv/2.2.6/developing/data-types/gsets.md]: Adjusted to reflect the banking context

**Pull request merged**